region/maridia/inner-pink.json

```diff
@@ -5109,7 +5381,22 @@
                   "notable": false,
                   "requires": [
                     "Morph",
-                    "Gravity"
+                    "Gravity",
+                    "canPlayInSand",
```

Comment:
I think we should try to keep it so that with all tech turned off and all items on, virtually all items should still be reachable. (Currently the only case that doesnt work is the shinespark item in Main Street)

with space jump + gravity, I would say this shouldn't require canplayinsand
Possibly also with hijump + walljump or hijump + springball midair, since the jump you get off the sand can be really terrible and still work